e2e/process-services-cloud/apps-section-cloud.e2e.ts

```diff
@@ -33,7 +33,10 @@ describe('Applications list', () => {
     const navigationBarPage = new NavigationBarPage();
     const settingsPage = new SettingsPage();
     const appListCloudPage = new AppListCloudPage();
-    const simpleApp = browser.params.resources.ACTIVITI7_APPS.SIMPLE_APP.name;
+    const simpleApp = browser.params.resources.ACTIVITI_CLOUD_APPS.SIMPLE_APP.name;
+    // tslint:disable-next-line:no-console
+    console.log('Stampo' + simpleApp);
```

Comment:
is this Italian? :)

Reply:
ahaha
yep just playing with the comment